### PR TITLE
Japanese translation

### PR DESCRIPTION
**Pull request by momo-i**
I used only forge, so missed fabric ver., now, I've been finished translate that.
BTW, I can't find messages for "gui.jade.configuration.desc1" and "gui.jade.configuration.desc2" on forge. where does use these messages?
I think from the source that these two messages are used concatenated (if there is no 2, only 1 is used), but is this part of the Forge version not used?

### CODE REVIEW

**Comment by Snownee** [comment]
Thank you!

Yes, the changes are not being ported to Forge yet. When you open the configuration screen on Fabric, you will immediately see it. if the sentence is too short compared to English, you can write the full sentence in desc1 and leave desc2 empty.

File: src/main/resources/assets/jade/lang/ja_jp.json [hunk 1]
```diff
@@ -27,16 +40,22 @@
   "narration.jade.negative": "否定 %s",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ja_jp.json]: this represents a negative number, so I think here is probably wrong

**Review by Snownee — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/jade/lang/ja_jp.json (hunk 1 above)
Comment (momo-i) [reply src/main/resources/assets/jade/lang/ja_jp.json]: OK, I checked this. FYI, where is this narration used? Is it read out loud when a number such as "-1" is entered or displayed?

Comment (Snownee) [reply src/main/resources/assets/jade/lang/ja_jp.json]: That's correct

Comment (momo-i) [reply src/main/resources/assets/jade/lang/ja_jp.json]: Fixed translation.

**Comment by Snownee** [comment]
Thanks again!

**Pull request merged**